og_ui/og_ui.module

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * @file
+ * Main functions and hook implementations of the Organic Groups UI module.
+ */
+
+use Drupal\Core\Config\Entity\ConfigEntityBundleBase;
+use Drupal\Core\Entity\BundleEntityFormBase;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\og\Og;
+use Drupal\og_ui\BundleFormAlter;
+
+/**
+ * Implements hook_form_alter().
+ */
+function og_ui_form_alter(array &$form, FormStateInterface $form_state, $form_id) {
+  if ($form_state->getFormObject() instanceof BundleEntityFormBase) {
+    (new BundleFormAlter($form_state->getFormObject()->getEntity()))
+      ->formAlter($form, $form_state);
+  }
+}
+
+/**
+ * Implements hook_entity_insert().
+ */
+function og_ui_entity_insert(EntityInterface $entity) {
+  og_ui_entity_type_save($entity);
+}
+
+/**
+ * Implements hook_entity_insert().
+ */
+function og_ui_entity_update(EntityInterface $entity) {
+  og_ui_entity_type_save($entity);
+}
+
+function og_ui_entity_type_save(EntityInterface $entity) {
+  if (!$entity instanceof ConfigEntityBundleBase || !isset($entity->og_is_group)) {
+    return;
+  }
+  $bundle = $entity->id();
```

Comment:
Can we add some comments and some spacing between the statements? Could be much more nice to read. 10x.